### PR TITLE
Fix TextInput issues on Android

### LINKED ISSUE #122: TextInput on Android have a large padding
67900608_454174445311560_3467196741508399104_n

68462402_351825639033470_5569365937619992576_n

68462402_351825639033470_5569365937619992576_n

### PR DESCRIPTION
**Pull request by NastuzziSamy**
Fix #122 and #123 
amount

### CODE REVIEW

File: src/components/Home/AmountInput.js [hunk 1]
```diff
@@ -12,20 +12,22 @@ import colors from '../../styles/colors';
...
 					color: error == null ? tintColor : colors.error,
 					alignSelf: 'center',
 					textAlign: 'right',
+					width: value ? 'auto' : 180,
+					padding: 0,
+					margin: 0,
 				}}
 				keyboardType="decimal-pad"
-				placeholder="00,00"
+				placeholder={value ? null : '00,00'}
```
Comment (aarrtthhuurr) [inline src/components/Home/AmountInput.js]: The default placeholder looks like this on iOS :
image

File: src/components/Home/AmountInput.js [hunk 2]
```diff
@@ -12,20 +12,22 @@ import colors from '../../styles/colors';
 
 export default function AmountInput({ value, error, maxLength, onChange, tintColor, autofocus }) {
 	return (
-		<View style={{ flexDirection: 'row' }}>
+		<View style={{ flexDirection: 'row', justifyContent: 'center' }}>
```
Comment (aarrtthhuurr) [inline src/components/Home/AmountInput.js]: Not a big fan of `justifyContent: 'center'`
Can someone else take a decision ?

File: src/components/Home/AmountInput.js (hunk 1 above)
Comment (NastuzziSamy) [reply src/components/Home/AmountInput.js]: Ok I go for `''` then

File: src/components/Home/AmountInput.js (hunk 2 above)
Comment (MercierCorentin) [reply src/components/Home/AmountInput.js]: I also use justifyContent with flexbox. Why are you not a big fan? Maybe there's something i don't know about it.

Comment (aarrtthhuurr) [reply src/components/Home/AmountInput.js]: Me too ahah, I was just saying that I prefer the amount left aligned than centered

**Comment by NastuzziSamy** [comment]
New force push

amount

**Pull request merged**